### PR TITLE
Downgrade PF

### PR DESCRIPTION
**Pull request by rawagner**
add basepath for all routes
create lib router component

### CODE REVIEW

File: src/components/Router.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { Switch, Route, Redirect } from 'react-router-dom';
+import { Clusters, ClusterPage } from './clusters';
+import { store } from '../store';
+import { routeBasePath } from '../config';
+
+export const BMRouter: React.FC = () => (
```
Comment (rawagner) [inline src/components/Router.tsx:8]: this will be used in uhc-portal

File: src/components/Router.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  <Provider store={store}>
+    <Router />
+  </Provider>
+);
+
+export const Router: React.FC = () => (
```
Comment (rawagner) [inline src/components/Router.tsx:14]: we can use this one in standalone app

**Comment by mareklibra** [comment]
Please do not merge yet. Stabilizing production.

**Comment by rawagner** [comment]
not needed anymore, uhc-portal is using newest PF now

**Pull request closed without merging**